resources/server_install.rb

```diff
@@ -73,15 +73,21 @@
   # Generate a random password or set a password defined with node['mariadb']['server_root_password'].
   # The password is set or change at each run. It is good for security if you choose to set a random password and
   # allow you to change the root password if needed.
+  set_password_command = "USE mysql;\n"
+  set_password_command += if new_resource.version =~ /^10\.[0-3]/
```

Comment:
This would be better I think.
```suggestion
  set_password_command += if new_resource.version.to_f <= 10.3
```

Reply:
Thanks for the suggestion, I have fixed the version comparison method